filebeat/input/event.go

```diff
@@ -18,9 +18,9 @@ type Event struct {
 	DocumentType string
 	Bytes        int
 	Text         *string
-	JSONFields   common.MapStr
 	JSONConfig   *reader.JSONConfig
 	State        file.State
+	Data         common.MapStr
```

Comment:
I didn't want to confuse "fields" which are set from the user with the "fields / data" added by the readers. I will add a doc comment.